### PR TITLE
🎨  patch projects when creating study job

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
Use `PATCH` method instead of `PUT` when changing name of study job from api-server.

## Related issue/s

- resolves [#5813](https://github.com/ITISFoundation/osparc-simcore/issues/5813)

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/models-library/src/models_library/basic_types.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class UUIDStr(ConstrainedStr):
 class IDStr(ConstrainedStr):
     strip_whitespace = True
     min_length = 1
-    max_length = 50
+    max_length = 100
```
Comment (matusdrobuliak66) [inline packages/models-library/src/models_library/basic_types.py:80]: Q: Why this change?

**Review by matusdrobuliak66 — APPROVED** [review]
Cool! thanks

File: packages/models-library/tests/test_basic_types.py [hunk 2]
```diff
@@ -72,4 +72,4 @@ def test_string_identifier_constraint_type():
     # limited to 50!
     parse_obj_as(IDStr, "X" * 50)
```
Comment (sanderegg) [inline packages/models-library/tests/test_basic_types.py:73]: don't you need to adapt that test here?

**Review by sanderegg — APPROVED** [review]

File: packages/models-library/tests/test_basic_types.py (hunk 2 above)
Comment (bisgaard-itis) [reply packages/models-library/tests/test_basic_types.py]: done

**Review by GitHK — APPROVED** [review]
